### PR TITLE
remove daisyUI

### PR DESCRIPTION
**Pull request by E-fais**
This PR fixes issue #25 
uninstalled daisyUI which is no more required in this project.

### CODE REVIEW

**Comment by E-fais** [comment]
@pushkaraj2007  please review..

**Comment by pushkaraj2007** [comment]
Looks like you are trying to remove the library manually, which you should not.

Do it with the help of `npm uninstall daisyui`

**Review by pushkaraj2007 — COMMENTED** [review]
It looks like you are trying to remove the library manually, which you should not.

Do it with the help of `npm uninstall daisyui`

**Comment by E-fais** [comment]
I have done it that way , in my local computer.

**Comment by E-fais** [comment]
What went wrong @pushkaraj2007

**Comment by pushkaraj2007** [comment]
Nothing @E-fais, but daisyui is still present in `package.json`

Can you please fix that?

**Comment by E-fais** [comment]
let me try again

**Review by E-fais — COMMENTED** [review]
@pushkaraj2007  can you remove my last commit.

**Comment by pushkaraj2007** [comment]
Why @E-fais?

**Comment by E-fais** [comment]
> Why @E-fais?

Ohh.. that was a typo , i was saying can you review my last commit , lol.

**Comment by pushkaraj2007** [comment]
Looks good 🔥

Thanks for contributing!! 🚀
